airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/activities/GenerateInputActivityImpl.java

```diff
@@ -111,7 +111,9 @@ public GeneratedJobInput getSyncWorkflowInput(final SyncInput input) {
           .withAttemptId((long) attempt)
           .withDockerImage(config.getDestinationDockerImage())
           .withProtocolVersion(config.getDestinationProtocolVersion())
-          .withIsCustomConnector(config.getIsDestinationCustomConnector());
+          .withIsCustomConnector(config.getIsDestinationCustomConnector())
```

Comment:
These changes look unrelated, maybe due to a merge/rebase with master?

Reply:
yep!